napari/layers/_shapes_layer/shapes/ellipse.py

```diff
@@ -61,7 +61,7 @@ def __init__(
 
     @property
     def data(self):
-        """np.ndarray: Nx2 array of vertices.
+        """np.ndarray: 4x2 array of vertices.
```

Comment:
shape in type

Reply:
so have no shape here, only `np.ndarray: array of vertices` and then shape up top 
```
Parameters
    ----------
    data : np.ndarray, 2x2 or 4x2
```
?

Reply:
`4x2` is a shape and it should be formatted as `(4, 2) array: ellipse vertices`

Reply:
fixed